dsc/tests/dsc_resource_test.tests.ps1

```diff
@@ -3,13 +3,20 @@
 
 Describe 'Invoke a resource test directly' {
     It 'test can be called on a resource' {
-        $os = if ($IsWindows) {
+        $os = if ($IsWindows)
```

Comment:
For PowerShell script, we keep the open brace on the same line to be consistent with Pester tests which require it